src/pkg/bundle/common.go

```diff
@@ -264,22 +263,15 @@ func ValidateBundleSignature(bundleYAMLPath, signaturePath, publicKeyPath string
 	return zarfUtils.CosignVerifyBlob(bundleYAMLPath, signaturePath, publicKeyPath)
 }
 
-// GetDeployedPackages returns packages that have been deployed
-func GetDeployedPackages() ([]zarfTypes.DeployedPackage, error) {
-	cluster := cluster.NewClusterOrDie()
-	deployedPackages, errs := cluster.GetDeployedZarfPackages()
-	if len(errs) > 0 {
-		return nil, lang.ErrUnableToGetPackages
-	}
-	return deployedPackages, nil
-}
-
 // GetDeployedPackageNames returns the names of the packages that have been deployed
 func GetDeployedPackageNames() []string {
 	var deployedPackageNames []string
-	deployedPackages, _ := GetDeployedPackages()
-	for _, pkg := range deployedPackages {
-		deployedPackageNames = append(deployedPackageNames, pkg.Name)
+	c, _ := cluster.NewCluster()
+	if c != nil {
+		deployedPackages, _ := c.GetDeployedZarfPackages()
```

Comment:
Unsolicited comment but I stumbled upon this PR because I ran into this bug today.

Should we explicitly check for the `err` here? It's unlikely, because of how we usually do variable returns in the Zarf library, but it is possible for us to get an err and a nil `deployedPackages` and run into a panic on the line below when we try to loop through it.

Reply:
I am realizing that my comment is technically wrong. You can range over a slice even if it `nil` without resulting in a panic.

Reply:
Good point though, we've gotten in trouble before by skipping these err checks. My thought with ignoring it here was that I don't really know what to do with the error; if there is an error, I think we want to continue the program not return the error